tests/unit/amo/components/TestAddon.js

```diff
@@ -44,7 +44,10 @@ import {
   dispatchSignInActions, fakeAddon, signedInApiState,
 } from 'tests/unit/amo/helpers';
 import {
-  createFetchAddonResult, getFakeI18nInst, sampleUserAgentParsed,
+  createFetchAddonResult,
+  getFakeI18nInst,
+  sampleUserAgentParsed,
+  createStubErrorHandler,
```

Comment:
This is a slightly different topic but have you considered [Prettier](https://github.com/prettier/prettier) to stop thinking about CS? (_à la_ `go fmt`)